README.md

```diff
@@ -8,6 +8,17 @@ for a rendered version of this material,
 for instructions on formatting, building, and submitting material,
 or run `make` in this directory for a list of helpful commands.
 
+## Run Jekyll in a Docker container
+
+If you download and install the latest versions of [Docker compose](https://docs.docker.com/compose/install/) and
+[Docker engine](https://docs.docker.com/engine/installation/) you can build a Jekyll container to serve this site with
+hot-reloading and incremental rebuilds by running `docker-compose up` and visiting `localhost:4000` in a browser. 
```

Comment:
Ah, good point. This is a shell command and also needs some testing on Windows and Mac, since I've only tested it on Linux.. I'll update the readme with more detailed instructions and update this PR.

Ultimately it'd be nice to have a script / tool that runs a Docker "sidecar" Jekyll container. This sidecar could then be attached to any of the software carpentry Jekyll lessons to build & serve the content without having to modify your base system (beyond installing Docker).